### PR TITLE
Adding support for ignoring ssl errors

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

File: Tasks/AzureResourceGroupDeployment/task.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 2,
-        "Patch": 11
+        "Patch": 12
     },
```
Comment (sachinma) [inline Tasks/AzureResourceGroupDeployment/task.json]: can you update the version of all the tasks consuming this package. And update the version of arm-rest..

Comment (ds-ms) [reply Tasks/AzureResourceGroupDeployment/task.json]: Done

File: Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts [hunk 2]
```diff
@@ -13,18 +13,22 @@ import utils = require("./Utils");
...
+    proxy: {
+        proxyUrl: proxyUrl,
+        proxyUsername: tl.getVariable("agent.proxyusername"),
+        proxyPassword: tl.getVariable("agent.proxypassword"),
+        proxyBypassHosts: tl.getVariable("agent.proxybypasslist") ? JSON.parse(tl.getVariable("agent.proxybypasslist")) : null
+    }
+} : {};
+
+let ignoreSslErrors = tl.getVariable("ignoreSslErrors");
+requestOptions.ignoreSslError = ignoreSslErrors ? ignoreSslErrors.toLowerCase() == "true" : false;
```
Comment (sachinma) [inline Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts]: you can say ignoreSslErrors && ingoreSslErrrors.ToLowerCase() == "true"

File: Tasks/Common/azure-arm-rest/package.json [hunk 3]
```diff
@@ -15,6 +15,6 @@
   "dependencies": {
     "q": "1.4.1",
     "vsts-task-lib": "2.0.5",
-    "typed-rest-client":"0.12.0"
```
Comment (sachinma) [inline Tasks/Common/azure-arm-rest/package.json]: Should you increase the version of the azure-arm-rest package

**Pull request merged**